graphql/schema.js

```diff
@@ -0,0 +1,46 @@
+const { gql } = require("apollo-server");
+
+const typeDefs = gql`
+  enum MovementTypes {
+    DEPOSIT
+    WITHDRAW
+  }
+
+  type User {
+    username: String!
+    id: ID!
+  }
+
+  type Token {
+    value: String!
+  }
+
+  type Account {
+    number: String!
+    balance: Int!
+    deposit: Int!
```

Comment:
Schema seems ok. We are guessing that you use `deposit` and `withdraw` as totalizers. No need for that I think :D

Reply:
OK, I'll see how I can do it without being an enum. :)